linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -1,193 +1,267 @@
-
-                        if [[ $OS_TYPE == "ubuntu" ]]; then
-                            PACKAGE_NAME="deb"
-                        elif [[ $OS_TYPE == "centos" ]]; then
-                            PACKAGE_NAME="rpm"
-                        fi
-
-                        pushd ${BUILD_NUMBER}${BRANCH_NAME}
-                        mkdir -p "$PACKAGE_NAME"
-                        art=$(find . -name "*.$PACKAGE_NAME")
-                        mv $art "$PACKAGE_NAME"
-                        popd
-                    '''
-                    sh '''#!/bin/bash
-                        OS_TYPE=${DISTRO_VERSION%_*}
-                        OS_TYPE=${OS_TYPE,,}
-                        local_path=$(readlink -f ${BUILD_NUMBER}${BRANCH_NAME})
-                        bash scripts/azure_kernel_validation/validate_azure_vm_boot.sh \
-                                --build_name $BUILD_NAME --build_number "${BUILD_NUMBER}${BRANCH_NAME}" \
-                                --smb_share_username $USERNAME --smb_share_password $PASSWORD \
-                                --smb_share_url $SMB_SHARE_URL --os_type $OS_TYPE --vm_user_name $OS_TYPE \
-                                --local_path $local_path
-                        '''
-                }
+            withCredentials(bindings: [string(credentialsId: 'SMB_SHARE_URL', variable: 'SMB_SHARE_URL'),
+                                              usernamePassword(credentialsId: 'smb_share_user_pass', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME')
+                                      ]) {
+                sh(script: """#!/bin/bash
+                              set -xe
+                              distro_version=$DISTRO_VERSION
+                              OS_TYPE=\${distro_version%_*}
+                              OS_TYPE=\${OS_TYPE,,}
+                              if [[ \$OS_TYPE == "ubuntu" ]]; then
+                                  PACKAGE_NAME="deb"
+                              elif [[ \$OS_TYPE == "centos" ]]; then
+                                  PACKAGE_NAME="rpm"
+                              fi
+                              pushd $BUILD_NUMBER$BRANCH_NAME
+                              mkdir -p "\$PACKAGE_NAME"
+                              art=\$(find . -name "*.\$PACKAGE_NAME")
+                              mv \$art "\$PACKAGE_NAME"
+                              popd
+                              """
+                )
+                exitStatus = sh(script: """#!/bin/bash
+                                           distro_version=$DISTRO_VERSION
+                                           OS_TYPE=\${distro_version%_*}
+                                           OS_TYPE=\${OS_TYPE,,}
+                                           local_path=\$(readlink -f ${BUILD_NUMBER}${BRANCH_NAME})
+                                           AZURE_MAX_RETRIES=$AZURE_MAX_RETRIES bash scripts/azure_kernel_validation/validate_azure_vm_boot.sh \
```

Comment:
all this script can be put with at least 4 spaces on the left

Reply:
line too long
everything that cannot be seen on github review page is definitely a line too long and it should be changed